@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -458,6 +411,85 @@ export class PoiManager {
         }
     }
 
+    /**
+     * Create and add POI [[TextElement]]s to tile with a series of positions.
+     */
+    private addPoi(
+        tile: Tile,
+        poiGeometry: PoiGeometry,
+        technique: PoiTechnique,
+        techniqueIdx: number,
+        positions: THREE.BufferAttribute
+    ) {
+        const poiTechnique = technique as PoiTechnique;
+        let imageTextureName = poiTechnique.imageTexture;
+        if (typeof poiTechnique.imageTexturePrefix === "string") {
+            imageTextureName = poiTechnique.imageTexturePrefix + imageTextureName;
+        }
+        if (typeof poiTechnique.imageTexturePostfix === "string") {
+            imageTextureName = imageTextureName + poiTechnique.imageTexturePostfix;
+        }
+
+        if (poiGeometry.stringCatalog === undefined) {
+            return;
```

Comment:
This condition should be at the start of the function. Otherwise, we execute unnecessary prefix/postfix code

Reply:
Again not my code, but ok, will fix.